archiecture/Motions.md

```diff
@@ -0,0 +1,123 @@
+
+So far, I've been referring to staking in the positive direction, or "for" a motion. It's also possible to stake "against" a motion, which we refer to as an "objection". An objection has the same threshold in order to be considered valid. 
+
+If a motion is not staked (positively) by 100% of the required stake, the motion will fail. 
+If a motion is staked 100%, and does not have an objection that is also staked 100%, the motion will pass. 
+
+These are the two simplest states of a motion outcome. Direct pass, and direct fail.
+
+In either of these cases, a motion will progress directly to the Finalize phase if it passed, or to the Claim phase if it failed. 
+
+The third case, in which a motion is both staked and objected 100%, triggers a Vote on the motion. In this case, the motion will immediately progress to the Voting phase, and not wait for the staking phase duration to elapse. 
+
+3. Voting. A vote is open to any wallet address with more than 0 reputation in the colony. You can either vote for the motion or against it. While still in the Voting phase, a user can change their vote as many times as they like. The voting phase will transition to the Reveal phase either when the Voting duration has elapsed, or when a certain minimum amount of reputation has voted on the motion (also specified when installing the Governance extension). That is to say, when the combined reputation of all the users that have voted on the motion reaches a certain threshold, the Voting phase may pass to Reveal without waiting for the duration to elapse.
+
+4. Reveal. This is where users reveal their votes to one another. Initially votes are hidden, and only in the reveal phase do users share them publicly. This is where the outcome of the motion is determined. Once all voters have revealed their votes, the side with the greatest number of votes wins, and the motion progresses to the Finalization phase. If not all voters reveal their votes, the motion will progress regardless once the Reveal phase duration has elapsed.
+
+5. Finalization. Anybody can finalize a motion, and it can only occur once. The result here depends on whether the motion passed or failed. If the motion failed, then as far as the users are concerned, nothing happens when a motion is finalized. However, if the motion passed, then the underlying on-chain action will be executed. Once a motion is finalized, it transitions to the Claim phase.
+
+6. The Claim phase is where anybody who staked the motion can have their stake returned. This applies whether the motion went to a vote or not. In the event a motion went to a vote, those on the winning side will receive a reward above their initial stake, which comes from the penalty applied to the stakes of those on the losing side. This incentivizes people to only stake / object motions that matter to them!
```

Comment:
How is the penalty calculated? Could you give an example here, similar the others above?

Reply:
Nein, I don't actually know